prototypes/entity.lua

```diff
@@ -1,302 +1,148 @@
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-1.2, -1.2}, {1.2, 1.2}},
+		selection_box = {{-1.5, -1.5}, {1.5, 1.5}},
+		fast_replaceable_group = "container",
+		inventory_size = 150,
+		picture =
+		{
+			filename = "__Warehousing__/graphics/entity/storehouse-basic.png",
+			priority = "high",
+			width = 129,
+			height = 100,
+			shift = {0.421875, 0},
+		},		
+		circuit_wire_max_distance = 7.5,
+		circuit_wire_connection_point =
+		{
+			shadow =
+			{
+				red = {.26, -0.6},
+				green = {.36, -0.6}
+			},
+			wire =
+			{
+				red = {-0.16, -0.9},
+				green = {0.16, -0.9}
+			}
+		},
+	},
+  {
+		type = "container",
+		name = "warehouse-basic",
+		icon = "__Warehousing__/graphics/icons/warehouse-basic.png",
+    icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "warehouse-basic"},
+		max_health = 450,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-2.7, -2.7}, {2.7, 2.7}},
+		selection_box = {{-3.0, -3.0}, {3.0, 3.0}},
+		fast_replaceable_group = "container",
+		inventory_size = 800,
+		picture =
+		{
+			filename = "__Warehousing__/graphics/entity/warehouse-basic.png",
+			priority = "high",
+			width = 260,
+			height = 240,
+			shift = {1.0, -0.3},
+		},		
+		circuit_wire_max_distance = 7.5,
+		circuit_wire_connection_point =
+		{
+			shadow =
+			{
+				red = {2.01, 0.6},
+				green = {2.52, 0.6}
+			},
+			wire =
+			{
+				red = {1.71, 0.3},
+				green = {2.22, 0.3}
+			}
+		},
+	},
+})
+
+-- generate logistic variants
+function createLogisticContainer(name, logistic_type)
+  local p = table.deepcopy(data.raw["container"][name.."-basic"])
+  p.name = name.."-"..logistic_type
+  p.minable.result = p.name
+  p.icon = "__Warehousing__/graphics/icons/"..p.name..".png"
+  p.picture.filename = "__Warehousing__/graphics/entity/"..p.name..".png"
+  p.type = "logistic-container"
+  p.logistic_mode = logistic_type
+  return p
+end
+
+local storehouse_active_provider = createLogisticContainer("storehouse", "active-provider")
+local storehouse_passive_provider = createLogisticContainer("storehouse", "passive-provider")
+local storehouse_storage = createLogisticContainer("storehouse", "storage")
+storehouse_storage.inventory_size = 300

```

Comment:
Storage logistic variants should have one logistic slot to allow filtering.
```suggestion
storehouse_storage.inventory_size = 300
storehouse_storage.logistic_slots_count = 1
```

Reply:
Yes, that pull was made before filter slots where a thing.